tasks/redhat.task/kickstart.erb

```diff
@@ -7,7 +7,7 @@ url --url=<%= repo_url %>
 cmdline
 lang en_US.UTF-8
 keyboard us
-rootpw <%= node.root_password %>
+rootpw <%= (node.root_password or '').gsub('\\\\', '\\\\'*4).gsub('#', '\\#').gsub('$', '\\$') %>
```

Comment:
```
#raw
<line>
#end raw
```
^^ looks like a more scalable and less finicky way to [handle this](https://access.redhat.com/documentation/en-US/Red_Hat_Network_Satellite/5.4/html/Getting_Started_Guide/sect-Red_Hat_Network_Satellite-Getting_Started_Guide-Kickstart-Templating.html#exam-Red_Hat_Network_Satellite-Getting_Started_Guide-Templating-Escaping_Special_Characters_in_templates).